apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -183,6 +183,11 @@ type WorkspaceConfig struct {
 	// the value "0" should be used. By default, the memory limit is 128Mi and the memory request is 64Mi.
 	// No CPU limit or request is added by default.
 	DefaultContainerResources *corev1.ResourceRequirements `json:"defaultContainerResources,omitempty"`
+	// ContainerResourceCaps defines the maximum resource requirements enforced for all
+	// workspace containers. If a container specifies higher limits or requests, they
```

Comment:
@tolusha, if a container does not specify limits / requests the maximums from DWOC will be used, right?
This is an important nuance that should be documented at the specification level, I believe.
@cgruver could you confirm that this is a requested behaviour?

Reply:
> if a container does not specify limits / requests the maximums from DWOC will be used, 

I would rather not have this functionality, since right now, `DefaultContainerResources` will be used instead which IMHO makes more sense